charmap.txt

```diff
@@ -0,0 +1,217 @@
+' ' = 20
```

Comment:
wait isn't this just ascii? why do we need a charmap for firered?

Reply:
Yeah, it's redundant. Oversight on my part, will fix.

Reply:
Unable to match extended ASCII with `gas` `.ascii`.